.github/workflows/deploy.yml

```
@@ -0,0 +1,38 @@
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

afaik we don't allow v1.2.3 type versions in the registry (must be 1.2.3 format). We may be allowing them now, but maybe change this to make the `v` optional
```suggestion
      - 'v?[0-9]+.[0-9]+.[0-9]+'
```